package.json

```
@@ -25,7 +25,7 @@
     "jest-environment-jsdom": "^29.7.0",
     "jest-localstorage-mock": "^2.4.26",
     "msw": "^2.6.9",
-    "prettier": "^3.6.2",
+    "prettier": "3.6.2",
```

I dont think you need to make this specifically 3.6.2, I would revert this back to the previous value.

```suggestion
    "prettier": "^3.6.2",
```

---

Will fix